### PR TITLE
Feature/ expanded story add and show Labels

### PR DESCRIPTION
**Pull request by kostadriano**
Refs: 
Labels (show): [#22721](https://central.cm42.io/projects/central-board-v2#story-22721)
Labels (add): [#23505](https://central.cm42.io/projects/central-board-v2#story-23505)

cm42 central 7

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -54,6 +54,7 @@ es:
   description: "Descripción"
   add user: "Agregar usuario"
   add new member: "Agregar un nuevo miembro"
+  add new label: "Añadir una nueva etiqueta"
```
Comment (abnersajr) [inline config/locales/es.yml]: Others  `add` cases are using the word `Agregar` why is that different? Would be better use the same.

File: config/locales/pt-BR.yml [hunk 2]
```diff
@@ -54,6 +54,7 @@ pt-BR:
   remove: "Remover"
   add user: "Adicionar usuário"
   add new member: "Adicionar um novo membro"
+  add new label: "Adicionar uma nova etiqueta"
```
Comment (abnersajr) [inline config/locales/pt-BR.yml]: `Etiqueta` sounds weird to me. Why not `tag`?

Comment (kostadriano) [reply config/locales/pt-BR.yml]: For me too, but we have this: `labels:  'Etiquetas'` in pt-BR translation, so I think we should change all at once in other PR.

Comment (abnersajr) [reply config/locales/pt-BR.yml]: LGTM 👍

File: config/locales/es.yml (hunk 1 above)
Comment (kostadriano) [reply config/locales/es.yml]: We have other cases to this Spanish translation, like: 

```
  add story: "Añadir articulo"
  add note: "Añadir nota"
  add task: 'Añadir tarea'
```

So, I was thinking that `Añadir` was being used to 'something' and `Agregar` to 'someone'. But searching I realized that both are synonymous.

File: app/assets/stylesheets/new_board/_react_tags.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  left: 0;
+  width: 100%;
+}
+
+.react-tags__suggestions ul {
+  margin: 4px -1px;
+  padding: 0;
+  list-style: none;
+  background: white;
+  border: 1px solid #D1D1D1;
```
Comment (wenderjean) [inline app/assets/stylesheets/new_board/_react_tags.scss]: What you think to move that color to `variables`?

File: app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryLabels.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    const tags = this.state.tags.filter((tag, tagIndex) => tagIndex !== index);
+
+    this.setState(
+      { tags }, () => {
+        onEdit(this.joinTags(this.state.tags))
+      }
+    );
+  }
+
+  handleAddition(tag) {
```
[the remark this answers is not in the document]
Comment (paulodiovani) [reply app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryLabels.js]: actually is 

```js
handleAddition = (tag) => {
   ...
} 
```

[the remark this answers is not in the document]
Comment (kostadriano) [reply app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryLabels.js]: Great idea, but our version of babel still does not support this syntax. We'll change this soon. :smile:

**Pull request merged**